src/applications/static-pages/homepage/HomepageSearch.jsx

```diff
@@ -73,6 +73,24 @@ const HomepageSearch = () => {
       e.target.value,
     )}&t=${false}`;
 
+    // Record the analytic event.
+    recordEvent({
+      event: 'view_search_results',
```

Comment:
- [ ] Verify homepage in-page event from Review Instance

Reply:
I'm getting the correct action label of "Homepage - Search"

<img width="720" alt="Screenshot 2022-11-30 at 2 46 16 PM" src="https://user-images.githubusercontent.com/732460/204904541-26d34d13-fd8a-4166-acfa-bbd959c48e5b.png">